### PR TITLE
Cleanup PoI details panel code

### PR DESCRIPTION
**Pull request by opyh**
The PoI details code has quite grown in the meantime and needed a cleanup before building on top of it. This change splits everything there into more readable chunks.

### CODE REVIEW

**Comment by opyh** [comment]
@mutaphysis I've rebased this, could you review + merge this so your new code is based on it? This contains is a new `renderPhotoSection()` method in `NodeToolbar.js` that probably is relevant for you :)

File: src/components/NodeToolbar/AccessibilityExtraInfo.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class AccessibilityExtraInfo extends React.Component<Props, State> {
 
 
 const StyledAccessibilityExtraInfo = styled(AccessibilityExtraInfo)`
-  margin-top: 1em;
+  margin-top: -10px !important;
```
Comment (mutaphysis) [inline src/components/NodeToolbar/AccessibilityExtraInfo.js]: 😭

File: src/components/NodeToolbar/IncentiveHint.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import styled from "../../../node_modules/styled-components";
+import colors from "../../lib/colors";
+
+const color = colors.editHintBackgroundColor;
+// const color = colors.linkColor;
+
+export const IncentiveHint = styled.span.attrs({ className: 'incentive-hint' })`
```
Comment (mutaphysis) [inline src/components/NodeToolbar/IncentiveHint.js]: I have the feeling that it is not only used for incentives?

File: src/components/PhotoUpload/PhotoUploadButton.js [hunk 3]
```diff
@@ -40,12 +41,12 @@ class PhotoUploadButton extends React.Component<Props, State> {
...
         className={`${className} link-button`}
-        aria-label={t`Upload images`} >
+        aria-label={t`Add images`} >
         <span className='button-icon'>
             <CameraIcon />
         </span>
-        <span className='button-label'>{t`Upload images`}</span>
-        <span className='motivation-hint'> {t`Your Good Deed!`}</span>
+        <span className='button-label'>{t`Add images`}</span>
+        <IncentiveHint>{t`Improve your karma!`}</IncentiveHint>
```
Comment (mutaphysis) [inline src/components/PhotoUpload/PhotoUploadButton.js]: Text changed

File: src/lib/generateMapsUrls.js [hunk 4]
```diff
@@ -30,17 +30,17 @@ export function generateMapsUrl(feature: Feature, placeName: string) {
 
   if (isBingMaps) {
     // translator: Button caption shown in the place toolbar
```
Comment (mutaphysis) [inline src/lib/generateMapsUrls.js]: -1: Remove translator caption & also below

File: src/components/PhotoUpload/PhotoUploadButton.js (hunk 3 above)
Comment (opyh) [reply src/components/PhotoUpload/PhotoUploadButton.js]: Intentionally, this is the original string. There are existing translations for this incl. English where the translated string is named 'your good deed!'. Silke and Svenja will work on English wording in the transifex interface so at some point we need a process to replace 'original' strings with their changes.

**Pull request merged**